sdk/core/azure-core-amqp/src/common/global_state.cpp

```diff
@@ -57,7 +57,8 @@ namespace Azure { namespace Core { namespace Amqp { namespace Common { namespace
     // information gets in the way of the message.
     if (logCategory == AZ_LOG_TRACE
         && (strcmp(func, "log_outgoing_frame") == 0 || strcmp(func, "log_incoming_frame") == 0
-            || strcmp(func, "log_message_chunk") == 0))
+            || strcmp(func, "log_message_chunk") == 0 || strcmp(func, "_log_outgoing_frame") == 0
+            || strcmp(func, "_log_incoming_frame") == 0))
```

Comment:
Why are some of these `_` prefixed and others are not? For exmaple, what is `_log_incoming_frame` vs `log_incoming_frame`?

Reply:
Different C compilers use different conventions for __FUNC__, which is where these values come from. 

I want to suppress the source level traces for the uAMQP `log_outgoing_frame` and `log_incoming_frame` functions since they do nothing but clutter up what would otherwise be rather straightforward logs.